### PR TITLE
Goliaths are gold core spawnable

### PR DESCRIPTION
**Pull request by ghost**
They're a powerful mob but probably fine since spiders exist in gold cores as well. The point of gold cores is spawning exotic mobs, and the roster is really small so maybe some spice here and there will do!

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 1]
```diff
@@ -31,6 +31,7 @@
 	var/pre_attack = 0
 	var/pre_attack_icon = "Goliath_preattack"
 	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide)
+	gold_core_spawnable = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: TRUE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
Fix yer damn master

**Comment by Armhulen** [comment]
what's a git

**Comment by nicbn** [comment]
1 line 13 commits

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Changelog

**Comment by XDTM** [comment]
Issue is that they break walls when walking; you should limit the wall-breaking to lavaland rock

**Comment by Armhulen** [comment]
XDTM they can't break plasters which xenobio containment has

**Comment by XDTM** [comment]
Even so, they have huge grief potential, and means you cannot contain gold core mobs inside a room without reinforced walls anymore, cause a goliath might spawn anytime

Someone's gonna release spiders because of that, i guarantee it

**Comment by XDTM** [comment]
Also i think they may be blacklisted from sentience potions, but i can't remember for sure

**Comment by Armhulen** [comment]
@XDTM they aren't but you should test it just in case!

**Comment by Armhulen** [comment]
but fair enough these are really strong, i have a better idea

**Pull request closed without merging**